src/main/java/core/basesyntax/sevrice/OperationExecutor.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.sevrice;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationExecutor {
+    void execute(FruitTransaction transaction);
```

Comment:
```suggestion
public interface OperationStrategy {
    OperationHanlder get(Operation operation);
```
lets implement a pattern Strategy here
this method should return an Instance of OperationHandler for the operation that is given

you should also create an interface OperationHandler that should have 1 method - handle(FruitTransaction transaction)
it should work similarly to your OperationService